### ISSUE TITLE
undefined function html?/0

### ISSUE THREAD

**Issue by sepow** [zanderxyz/veil#12]
Just saw your post about this project on the elixir forums and thought I'd give it a go. 

Following the install instructions using phoenix 1.3.3 gives me this compile error though.

== Compilation error in file lib/testin/veil/veil.ex ==
** (CompileError) lib/testin/veil/veil.ex:62: undefined function html?/0
    (stdlib) lists.erl:1338: :lists.foreach/2
    (stdlib) erl_eval.erl:677: :erl_eval.do_apply/6
    (elixir) lib/kernel/parallel_compiler.ex:198: anonymous fn/4 in Kernel.ParallelCompiler.spawn_workers/6

**Comment by zanderxyz** [comment]
Hi Sepow, this was a dumb mistake on my side, I will fix now. You can delete lines 62, 64, 65 and 66 and it will work.
```
cur_path = <%= web_module %>.Router.Helpers.session_path(conn, :create, unique_id)
```

**Comment by zanderxyz** [comment]
So the function in question will just look like this:

```
defp new_session_url(conn, unique_id) do
    cur_uri = Phoenix.Controller.endpoint_module(conn).struct_url()
    cur_path = <%= web_module %>.Router.Helpers.session_path(conn, :create, unique_id)
    <%= web_module %>.Router.Helpers.url(cur_uri) <> cur_path
  end
```

**Comment by zanderxyz** [comment]
Now fixed in 0.1.6
